### PR TITLE
Piriform.CCleanerpro version 5.80

### PR DESCRIPTION
**Pull request by mddtest**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16413)

### CODE REVIEW

File: manifests/p/Piriform/CCleanerpro/5.80/Piriform.CCleanerpro.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate
+
+PackageIdentifier: Piriform.CCleanerpro
+PackageVersion: 5.80
+PackageLocale: en-US
+Publisher: Piriform
+PublisherUrl: https://www.ccleaner.com
+PublisherSupportUrl: https://support.piriform.com/hc/en-us
+Author: Piriform Software Ltd
+PackageName: CCleaner Professional Trial
```
Comment (OfficialEsco) [inline manifests/p/Piriform/CCleanerpro/5.80/Piriform.CCleanerpro.locale.en-US.yaml:10]:
```suggestion
PackageName: CCleaner
```

**Review by OfficialEsco — COMMENTED** [review]
Hmmm, this might cause a issue since both Normal and Pro have the same name, why don't they actually add CCleaner Pro to the name?..
Might be worth a try to see if it remembers if you installed Normal or Pro

File: manifests/p/Piriform/CCleanerpro/5.80/Piriform.CCleanerpro.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Piriform.CCleanerpro
```
Comment (ItzLevvie) [inline manifests/p/Piriform/CCleanerpro/5.80/Piriform.CCleanerpro.yaml:3]:
```suggestion
PackageIdentifier: Piriform.CCleanerPro
```

File: manifests/p/Piriform/CCleanerpro/5.80/Piriform.CCleanerpro.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate
+
+PackageIdentifier: Piriform.CCleanerpro
```
Comment (ItzLevvie) [inline manifests/p/Piriform/CCleanerpro/5.80/Piriform.CCleanerpro.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: Piriform.CCleanerPro
```

File: manifests/p/Piriform/CCleanerpro/5.80/Piriform.CCleanerpro.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate
+
+PackageIdentifier: Piriform.CCleanerpro
```
Comment (ItzLevvie) [inline manifests/p/Piriform/CCleanerpro/5.80/Piriform.CCleanerpro.installer.yaml:3]:
```suggestion
PackageIdentifier: Piriform.CCleanerPro
```

**Comment by ItzLevvie** [comment]
You'll also have to rename the folder and files to fix the capitalization issue.

**Comment by mddtest** [comment]
> 
> 
> You'll also have to rename the folder and the file names to fix the capitalization issue using the GitHub Desktop client on Windows.

was just doing that

**Pull request closed without merging**